openlibrary/catalog/marc/fast_parse.py

```diff
@@ -200,14 +181,31 @@ def get_tag_line(data, line):
             length += data[last:].find('\x1e')
     except IndexError:
         pass
-
     tag_line = data[offset + 1:offset + length + 1]
     if not line[0:2] == '00':
-        # marc_western_washington_univ/wwu_bibs.mrc_revrev.mrc:636441290:1277
         if tag_line[1:8] == '{llig}\x1f':
             tag_line = tag_line[0] + u'\uFE20' + tag_line[7:]
     return tag_line
 
+def get_tag_lines(data, want):
```

Comment:
Why did this functions get reordered? It makes it tough to review.

Reply:
I was moving the similar functions in the module that were used together so the marc/fast_parse.py module was easier to understand at a glance.